libbeat/docs/filteringconfig.asciidoc

```diff
@@ -13,55 +13,191 @@
-exported by the Beat. See <<exported-fields>> for the full list of possible fields.
+You can set a list of filter actions in the `filter` section of the {beatname_uc} config file to reduce the number of fields that are
+exported by the Beat. 
 
-The filter actions that you specify are applied to each event before it's sent to the defined output. You can define
-multiple filter actions. The filter actions are executed in the order that they're defined in the config file.
+If more filtering rules are defined, then they are executed in the order they are defined. The initial event is passed to the first filtering rule and what results from it is passed to the second filtering rule until all the filtering rules are applied. The condition that is used in the following filtering rules is running against the event that is received as input and it might defer from the original event.
 
-==== Filter Options
+[source,yaml]
+-------
+event -> filter action 1 -> event1 -> filter action 2 -> event2 ...
+-------
+
+See <<exported-fields>> for the full list of possible fields.
+
+Each filter action receives a condition and optionally a set of arguments. The action is executed only if the condition
+is fulfilled.
+
+[source,yaml]
+------
+filter:
+ - action1:
+     condition1
+     [arguments]
+ - action2:
+     condition2
+	 [arguments]
+...
+
+------
+
+See <<filtering-data>> for specific {beatname_uc} examples.
+
+[[filtering-condition]]
+==== Filtering condition
+
+Each condition receives a field to compare or multiple fields under the same condition and then `AND` is used between
+them. You can see a list of the <<exported-fields,`exported fields`>>. 
+
+For each field, you can specify a simple field name or a nested map, for example `dns.question.name`.
+
+[source,yaml]
+----
+condition:
+  field1: value1
+  [field2: value2]
+  ...
+----
+
+Supported conditions are:
+
+* <<condition-equals,`equals`>>
+* <<condition-contains,`contains`>>
+* <<condition-regexp,`regexp`>>
+* <<condition-range, `range`>>
+
+
+[[condition-equals]]
+===== equals
+
+With the `equals` condition, you can compare if a field have a certain value. It accepts only an integer or a string
+value.
+
+For example, the following condition checks if the response code of the HTTP transaction is 200:
+
+[source,yaml]
+-------
+equals:
+  http.code: 200
+-------
+
+[[condition-contains]]
+===== contains
+
+The `contains` condition checks if value is part of the field. It accepts only a string as value.
+
+For example, the following condition checks if an error is part of the transaction status:
+
+[source,yaml]
+------
+contains:
+  status: "Specific error"
+------
+
+
+[[condition-regexp]]
+===== regexp
+
+The `regexp` condition checks the field against a regular expression. It accepts only strings.
+
+For example, the following condition checks if the process name has a certain format:
+
+[source,yaml]]
+-----
+reqexp:
+  proc.name: 58gdhsga-=kw+++
```

Comment:
Good question. It's from our tests. I think I should replace with something more "readable" :)